### PR TITLE
Add files via upload

### LINKED ISSUE #24: Create the Department Model
See #10 

Feel free to assign yourself!

### PR DESCRIPTION
**Pull request by btgathright**
Closes #24 .

### CODE REVIEW

**Comment by gannonprudhomme** [comment]
Did you commit this by dragging & dropping the files into GitHub?

**Comment by btgathright** [comment]
I looked for the file directory on my computer and attached it to the
branch and pull request. How do I submit a pull request that doesn’t
overwrite the existing file until y’all approve it?

**Comment by btgathright** [comment]
For a description should we just put a commented block of what the model is or is it actual code?

**Comment by gannonprudhomme** [comment]
> For a description should we just put a commented block of what the model is or is it actual code?

The description for a pull request should basically just be what the pull request does/is achieving, but in more detail. In a situation like this where it's pretty straightforward, the PR title could be something like `Implemented department model`, and the description could basically contain any general concerns/questions you had about it. 

Likewise, if the PR completely resolves the issue that its for(which this one does), you can add `Closes #issuenumber` to the bottom of the description - which I did for you just now.

**Comment by btgathright** [comment]
I meant description in my departments.py file. There’s a line added that
shows I need a description of the model. Does this mean that I add comments
describing what it does or do I need to add more stuff to it?

**Comment by gannonprudhomme** [comment]
> I looked for the file directory on my computer and attached it to the branch and pull request. How do I submit a pull request that doesn’t overwrite the existing file until y’all approve it?

Is this your first time using Git? If so, check out the [Getting started with git](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/wiki/Getting-started-with-git) wiki page.

But to answer your question, you do so by creating a branch and checkout'ing it. Then when you're ready, you create a Pull request like you did here to merge it to master/backend.

Since these changes are on your own branch, you can ensure that you're not affecting anyone else's work.

File: autoscheduler/scraper/models/department.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
 from django.db import models
 
 class Department(models.Model):
-    """ Describe model here """
+    code = models.CharField(max_length=4, primary_key=True) # Ex: CSCE
+    description = models.TextField(max_length=100) # Ex: CSCE - Computer Sci & Engr
+
+    class Meta:
+        db_table = "departments"
```
Comment (gannonprudhomme) [inline autoscheduler/scraper/models/department.py]: Should have a new line at the end of it. Do you have pylint set up?

**Comment by gannonprudhomme** [comment]
> I meant description in my departments.py file. There’s a line added that shows I need a description of the model. Does this mean that I add comments describing what it does or do I need to add more stuff to it?

Oooh ok my bad. So that's typically referred to as the "docstring"(at least that's what pylint calls it). In this case since it's pretty straightforward so I'm not even sure it needs one, although I know pylint is going to yell at you to include one. For now, I'd just say like "Represents a department object, such as CSCE" or something along those lines.

**Comment by gannonprudhomme** [comment]
Also another thing, should we rename this from Department to Subject? It's generally referred to as subject elsewhere inside of Howdy/Banner

**Comment by btgathright** [comment]
I'm not sure what the docstring line you're asking to add is. I was using the models you had as templates from a previous scraper but don't see a line like that.

File: autoscheduler/scraper/models/department.py [hunk 2]
```diff
@@ -1,4 +1,8 @@
 from django.db import models
 
 class Department(models.Model):
-    """ Describe model here """
```
Comment (gannonprudhomme) [inline autoscheduler/scraper/models/department.py]: This is the docstring line

**Pull request closed without merging**